data-formats/src/constants/mod.rs

```diff
@@ -425,15 +425,15 @@ pub enum ErrorCode {
 #[repr(i8)]
 #[non_exhaustive]
 pub enum MfgStringType {
-    SerialNumber = 0,
+    SerialNumberExt = 0,
 }
 
 impl FromStr for MfgStringType {
     type Err = Error;
 
     fn from_str(s: &str) -> Result<Self> {
         Ok(match &s.to_lowercase()[..] {
-            "serialnumber" | "serial_number" => MfgStringType::SerialNumber,
+            "serialnumber" | "serial_number" => MfgStringType::SerialNumberExt,
```

Comment:
I think _changing_ this as opposed as _adding_ a new config option would be best - otherwise we risk people finding out we introduced new unexpected behavior for some

Reply:
taken care in latest commit , now user can specify SerialNumber or SerialNumberExt value for mfg_string_type in manufacturing_server.yml , which will decide existing(old) or the new format for serial number